ui/components/app/modals/deposit-token-modal/deposit-token-modal.container.js

```diff
@@ -0,0 +1,59 @@
+import { connect } from 'react-redux';
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props';
+import {
+  getCurrentChainId,
+  getIsBuyableCoinbasePayChain,
+  getIsMainnet,
+  getIsTestnet,
+  getSelectedAddress,
+  getIsBuyableTransakChain,
+  getIsBuyableCoinbasePayToken,
+  getIsBuyableTransakToken,
+} from '../../../../selectors/selectors';
+import {
+  buyEth,
```

Comment:
I was thinking if it will be more contextual to create another action creator called `buyToken` instead of using `buyEth`

Reply:
after our talk, we agreed to create a duplicate `buyToken` function for the sake of context clarity. choices were:

- `buyEth` for ETH and Tokens (contextually confusing)
- rename `buyEth` to `buyAssets` (possibility of impacting code in unexpected ways)
- create a `buyToken` function (selected)